var/spack/repos/builtin/packages/process-in-process/package.py

```diff
@@ -27,157 +37,167 @@ class ProcessInProcess(Package):
-        git="https://github.com/RIKEN-SysSoft/PiP-gdb.git",
-        branch="centos/gdb-7.6.1-94.el7.pip.branch",
-        destination="PiP-gdb",
-        when="@2 os=centos7",
-    )
-    resource(
-        name="PiP-gdb",
-        git="https://github.com/RIKEN-SysSoft/PiP-gdb.git",
-        branch="centos/gdb-7.6.1-94.el7.pip.branch",
-        destination="PiP-gdb",
-        when="@2 os=rhel7",
-    )
-    #   for rhel/centos 8
-    resource(
-        name="PiP-gdb",
-        git="https://github.com/RIKEN-SysSoft/PiP-gdb.git",
-        branch="centos/gdb-8.2-12.el8.pip.branch",
-        destination="PiP-gdb",
-        when="@2 os=centos8",
-    )
-    resource(
-        name="PiP-gdb",
-        git="https://github.com/RIKEN-SysSoft/PiP-gdb.git",
-        branch="centos/gdb-8.2-12.el8.pip.branch",
-        destination="PiP-gdb",
-        when="@2 os=rhel8",
-    )
-
-    # resources for PiP version 3
-    #  PiP-glibc resource
-    #   for rhel/centos 7
-    resource(
-        name="PiP-glibc",
-        git="https://github.com/RIKEN-SysSoft/PiP-glibc.git",
-        branch="centos/glibc-2.17-260.el7.pip.branch",
-        destination="PiP-glibc",
-        when="@3 os=centos7",
-    )
-    resource(
-        name="PiP-glibc",
-        git="https://github.com/RIKEN-SysSoft/PiP-glibc.git",
-        branch="centos/glibc-2.17-260.el7.pip.branch",
-        destination="PiP-glibc",
-        when="@3 os=rhel7",
-    )
-    #   for rhel/centos 8
-    resource(
-        name="PiP-glibc",
-        git="https://github.com/RIKEN-SysSoft/PiP-glibc.git",
-        branch="centos/glibc-2.28-72.el8_1.1.pip.branch",
-        destination="PiP-glibc",
-        when="@3 os=centos8",
-    )
-    resource(
-        name="PiP-glibc",
-        git="https://github.com/RIKEN-SysSoft/PiP-glibc.git",
-        branch="centos/glibc-2.28-72.el8_1.1.pip.branch",
-        destination="PiP-glibc",
-        when="@3 os=rhel8",
-    )
-
-    #  PiP-gdb resource
-    #   for rhel/centos 7
-    resource(
-        name="PiP-gdb",
-        git="https://github.com/RIKEN-SysSoft/PiP-gdb.git",
-        branch="centos/gdb-7.6.1-94.el7.pip.branch",
-        destination="PiP-gdb",
-        when="@3 os=centos7",
-    )
-    resource(
-        name="PiP-gdb",
-        git="https://github.com/RIKEN-SysSoft/PiP-gdb.git",
-        branch="centos/gdb-7.6.1-94.el7.pip.branch",
-        destination="PiP-gdb",
-        when="@3 os=rhel7",
-    )
-    #   for rhel/centos 8
-    resource(
-        name="PiP-gdb",
-        git="https://github.com/RIKEN-SysSoft/PiP-gdb.git",
-        branch="centos/gdb-8.2-12.el8.pip.branch",
-        destination="PiP-gdb",
-        when="@3 os=centos8",
-    )
-    resource(
-        name="PiP-gdb",
-        git="https://github.com/RIKEN-SysSoft/PiP-gdb.git",
-        branch="centos/gdb-8.2-12.el8.pip.branch",
-        destination="PiP-gdb",
-        when="@3 os=rhel8",
-    )
+    if enable_pip_gdb:
+        #  PiP-gdb resource
+        #   for rhel/centos 7
```

Comment:
Same request here

Reply:
You may know that GDB requires ncurses but ncurses cannot be installed by  using Spack (I checked and there was a discussion on this in the Spack slack. The problem is that ncurses tries to invoke ldconfig for users to use the installed ncurses, but this is not allowed by Spack.) 
I would appreciate it if you let me know a proper way to enabling/disabling PiP-gdb (patched GDB) so that I will enable PiP-gdb when the ncurses isuues is fixed.

Reply:
Well, now it seems that 'spack sty;e --fix' works. So close this PR and create a new one (#32347).